tests/sentry/api/endpoints/test_auth_index.py

```diff
@@ -29,7 +29,7 @@ def test_valid_password(self):
         response = self.client.post(
             self.path,
             HTTP_AUTHORIZATION=u"Basic {}".format(
```

Comment:
What does HTTP_AUTHORIZATION need to be? You can't mix, it needs to be either bytes or unicode for both the format string and arguments. Otherwise you end up with like, this in py3:

```
>>> u"ayy{}".format(b"lmao")
"ayyb'lmao'"
```

Reply:
Oh wait, python3 doesn't even let you have a bytes format string:

```
>>> b"ayy{}".format(b"lmao")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'bytes' object has no attribute 'format'
```

Disregard me here, I didn't see that you changed to `b"Basic " +` which isn't a format string. But it should be `u`.

Reply:
Ooh, yeah good catch. It probably will take either tbh

Reply:
I think you want `u"foo {}".format(force_text(b64encode(force_bytes(u"bar {}".format(u"baz")))))`, lol.

b64encode wants bytes, formatting requires a unicode format string in py3 and unicode arguments.

Reply:
That's if you want unicode for `HTTP_AUTHORIZATION`. If you want bytes then you must wrap that in another force_bytes, or not use format and just concatenate `b"foo " + b64encode(...)`. I think format is pretty painful because format string and arguments all need to be unicode, though py3 permits bytes arguments but they will have the literal `b` which chances are you don't want.